### PR TITLE
Updated Ducky Template for firmware version 1.1

### PR DESCRIPTION
**Pull request by Biocow**
Updated version number.
Updated LED status table.

### CODE REVIEW

File: payloads/library/general/DuckyTemplate/payload.txt [hunk 1]
```diff
@@ -1,18 +1,19 @@
-LED G
+LED SETUP
 ATTACKMODE HID STORAGE
 
 # Set your language here
-LANGUAGE='us'
+DUCKY_LANG=us
```
Comment (sebkinne) [inline payloads/library/general/DuckyTemplate/payload.txt]: This should be `DUCK_LANG us` instead as you are calling the DUCK_LANG extension and not trying to set a variable yourself.

**Comment by sebkinne** [comment]
@Biocow if you fix the issue above, I'd be happy to accept this PR.

**Comment by Biocow** [comment]
@sebkinne Thanks for catching that. Fixed.

**Comment by Biocow** [comment]
I thought you were saying it should be DUCK_LANG and not DUCKY_LANG, but after looking at the docs I see what you meant. DUCKY_LANG is a function that accepts a string (us) vs. using an equals sign and assigning a value. Think it should be good now.

**Comment by sebkinne** [comment]
@Biocow you are right - my phone must have autocorrected the command :(

**Pull request merged**